product_landing_page/index.css

```diff
@@ -0,0 +1,81 @@
+body {
+  background-color: #eee;
+  margin: 0;
+  font-family: 'Montserrat', sans-serif;
+
+}
+
```

Comment:
Warto też dodać taki selektor:
```
* {
    box-sizing: border-box;
}
```

Poczytaj sobie jaką to różnicę robi i postaraj się napisać w odpowiedzi jak to zrozumiałeś :) Być może jakaś część layoutu będzie do zmiany po tym.

Reply:
Rozumiem to w ten sposób, że właściwość ta powoduje określenie rzeczywistego rozmiaru diva, czyli jak dodaje paddingi czy bordery to rozmiar mojego diva zawierającego tę właściwość zmniejsza się o wartość paddingów/borderów